volumes/frus1883.xml

```diff
@@ -72629,7 +72629,7 @@
                                         <persName type="to">Aristarchi Bey</persName>.</hi></head>
                             <opener><dateline rendition="#right"><placeName><hi rend="smallcaps"
                                             >Sublime Porte</hi>,<lb/><hi rend="italic">Ministry of
-                                            Foreign Affairs</hi>.</placeName> [<date
+                                            Foreign Affairs</hi></placeName>[, <date
```

Comment:
Please confirm the spacing here (if the website can't open the link, it's line 72632 of frus1883).

Reply:
Correct. See above.